lib/resourceapply/machineconfig.go

```diff
@@ -80,21 +81,32 @@ func ApplyMachineConfigNode(client mcfgclientalphav1.MachineConfigNodesGetter, r
 
 // ApplyControllerConfig applies the required machineconfig to the cluster.
 func ApplyControllerConfig(client mcfgclientv1.ControllerConfigsGetter, required *mcfgv1.ControllerConfig) (*mcfgv1.ControllerConfig, bool, error) {
+	klog.Info("Getting existing ControllerConfig with name: ", required.GetName())
```

Comment:
suggestion (non-blocking): You can use `klog.Infof()` here for better control over what the resulting log entry looks like.

Reply:
suggestion (non-blocking): We may not want to log this every time because it would lead to a lot of additional logs being written. Instead, we may want to put this behind a `klog.V(4)` instead so that it's only enabled whenever `-v4` is given at startup.